testsuite/features/secondary/proxy_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) 2021 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@proxy
+@private_net
+@pxeboot_minion
+@scope_cobbler
+Feature: PXE boot a terminal with Cobbler
+  In order to automate client system installations in SUSE Manager
+  As the system administrator
+  I want to PXE boot one host with Cobbler
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Configure PXE part of DHCP on the proxy
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Formulas" in the content area
+    And I follow first "Dhcpd" in the content area
+    And I enter the local IP address of "proxy" in pxeboot next server field
+    And I enter "pxelinux.0" in pxeboot filename field
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Configure PXE part of DNS on the proxy
+    Given I am on the Systems overview page of this "proxy"
+    When I follow "Formulas" in the content area
```

Comment:
Are you sure you need to go back to proxy overview page and then re-access to Formulas tab?

Reply:
I believe is save to drop all but the first one.